### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by xzhxzhxzhxzh** [whubaichuan/M3VSNet#12]
hello,why is my initial loss around 5.

**Comment by whubaichuan** [comment]
Hi, can you put the whole training process here?

**Comment by xzhxzhxzhxzh** [comment]
![屏幕截图 2024-09-08 141140](https://github.com/user-attachments/assets/03901c1b-2c29-4223-a65c-108144ca0858)
![屏幕截图 2024-09-08 141122](https://github.com/user-attachments/assets/3e7c2b9d-eb9b-4311-a7cd-46f4b832c45a)

**Comment by xzhxzhxzhxzh** [comment]
北方树叶
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "whubaichuan/M3VSNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月8日(星期天) 下午2:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [whubaichuan/M3VSNet] loss (Issue #12)





 
Hi, can you put the whole training process here?

**Comment by xzhxzhxzhxzh** [comment]
Could it be a problem with the environment

**Comment by whubaichuan** [comment]
hi, thanks. can you try to finish the 10 epochs (10*6774)? I think the initial loss will not influence the final results, as you worry about.

In some cases, the real training process can not be the same as the perfect-tutorial training loss curve, such as [it](https://developers.google.com/machine-learning/glossary?hl=zh-cn#loss_curve). In real training, usually, it fluctuates.

**Comment by xzhxzhxzhxzh** [comment]
您好，其实之前我已经训练了十轮，不过我是在特征提取模块处又加了个注意力机制，它的损失函数也变化不大。

**Comment by whubaichuan** [comment]
Hi, thanks, can you get similar final results (ACC, COMP, and OVERALL) as presented in the readme.md, with the no-modified code?

**Comment by xzhxzhxzhxzh** [comment]
您好,我有一些问题想请教您.我现在的研究方向是关于三维重建的,但是老师关于这方面是零基础.但是他申请了一个关于三维重建的横向课题,让我们自建数据集进行迁移学习.但是今年已经研三了还没有搞定,我就想在您的网络的基础上进行修改,然后争取可以毕业.请问您有啥建议吗,关于网络模型修改方面的,谢谢.




北方树叶
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "whubaichuan/M3VSNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月8日(星期天) 下午3:52
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [whubaichuan/M3VSNet] loss (Issue #12)





 
Hi, thanks, can you get similar final results (ACC, COMP, and OVERALL) as presented in the readme.md, with the no-modified code?

**Comment by xzhxzhxzhxzh** [comment]
您好,另外我关注到好像做mvs的无论有监督还是无监督好像都比较难,或者说做的人很少,另外我关注到最近关于无监督最新的研究是cl-mvsnet,您是建议在最新的研究上改进(感觉有点难),还是在您的研究基础上改进呢,谢谢!


北方树叶
***@***.***



&nbsp;

**Comment by xzhxzhxzhxzh** [comment]
您好，我还没有在不修改代码的情况下训练，后面我训练一下。谢谢您
